packages/api/src/format/input.ts

```diff
@@ -5,7 +5,7 @@
 
 import BigNumber from 'bignumber.js';
 
-import { BlockNumber } from '../types';
+import { BlockNumber, InputDeriveHashMap, InputOptions, InputOptionsConditions, InputTrace, InputTraceHashMap } from '../types';
```

Comment:
I prefer in this PR to not touch anything in `packages/api`. This package needs some bigger effort to make it clean, see #21.